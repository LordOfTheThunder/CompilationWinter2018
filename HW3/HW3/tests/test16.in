struct t {
	int p;
	bool s;
	int x;
	int t;
};
void func() {
	int p;
}



void main() { print("I hate this course"); }

