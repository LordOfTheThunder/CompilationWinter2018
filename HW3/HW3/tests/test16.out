---end scope---
p INT 0
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
struct t {INT p,BOOL s,INT x,INT t}
func ()->VOID 0
